tests/test_formatters.py

```diff
@@ -982,6 +983,15 @@ def test_strip_whitespace(value):
     assert strip_whitespace(value) == 'bar'
 
 
+@pytest.mark.parametrize('value', [
+    'notifications-email',
+    '  \tnotifications-email \x0c ',
+    '\rn\u200Coti\u200Dfi\u200Bcati\u2060ons-\u180Eemai\uFEFFl\uFEFF',
```

Comment:
Is it worth testing a counter example? i.e. that a word with normal spaces or newlines doesn’t have those spaces removed.

Reply:
👍 I've added a test to check that 'normal spaces' get removed from the ends, but not the middle of a string

Reply:
👍